sdk/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -838,6 +838,10 @@ void notify(@NonNull Error error,
         // Attach user info to the error
         error.setUser(user);
 
+        // Attach default context from active activity
+        String context = config.getContext();
+        error.setContext(context != null ? context : appData.getActiveScreenClass());
```

Comment:
This needs to check whether the context has already been set on the error object, and avoid setting the field if this is the case. For example, the context will be wrong for the following `notify` method on `Client`:

```
Bugsnag.getClient().notify("Test", "exc_message", "CustomContext", new StackTraceElement[]{}, Severity.ERROR, new MetaData());
```